File: GTTS/INL.PersonService.Models/GetMatchingPersons_Result.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.PersonService.Models
{
    public class GetMatchingPersons_Result : IGetMatchingPersons_Result
    {
        public List<GetMatchingPersons_Item> MatchingPersons { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/get-matching-persons_result.ts


import { GetMatchingPersons_Item } from './get-matching-persons_item';
import { IGetMatchingPersons_Result } from './iget-matching-persons_result';

export class GetMatchingPersons_Result implements IGetMatchingPersons_Result {
  
	public MatchingPersons?: GetMatchingPersons_Item[];
  
}


